### ISSUE TITLE
Compiling dvi -> ps -> pdf

### ISSUE THREAD

**Issue by dpo** [andrewrynhard/atom-latex-plus#29]
Certain older style files and packages don't play very well with `pdflatex` and still require that you use the `latex`/`dvips`/`ps2pdf` sequence. I'm having trouble getting `atom-latex-plus` to honor a local `.latexmkrc`. Here's `test.tex`:

``` tex
\documentclass[10pt]{article}

\begin{document}
  This is a \LaTeX\ document.
\end{document}
```

My config file is

``` json
{
  "projectPath": "/private/tmp",
  "title": "Test Project",
  "root": "test.tex",
  "program": "pdflatex",
  "output": ".latex"
}
```

and there's a `.latexmkrc` in the same folder as `test.tex`:

``` rc
$pdf_mode = 2;
$latex = 'latex -file-line-error -shell-escape -synctex=1 %O %S';
$pdflatex = 'pdflatex -file-line-error -shell-escape -synctex=1 %O %S';
$dvips = "dvips %O -o %D %S";
$dvips_landscape = "dvips -tlandscape %O -o %D %S";
$dvips_pdf_switch = "-P pdf";
$ps2pdf = "ps2pdf %O %S %D";
$bibtex_use = 2;
```

At the command line, `latexmk test` generates `test.dvi`, `test.ps` and `test.pdf`. But building `test.tex` inside Atom doesn't:

```
$ ls .latex
test.aux         test.fls         test.pdf
test.fdb_latexmk test.log         test.synctex.gz
```

How do I get `atom-latex-plus` to honor my `.latexmkrc`?

**Comment by andrewrynhard** [comment]
@dpo currently it does not but this certainly can be implemented. According to http://users.phys.psu.edu/~collins/software/latexmk-jcc/latexmk-304.txt, the rc file in the same directory should not start with a dot:

```
       1)  The  system  RC file.  On a UNIX system, it may be any
       one of the  following  "/opt/local/share/latexmk/LatexMk",
       "/usr/local/share/latexmk/LatexMk",
       "/usr/local/lib/latexmk/LatexMk".  On a MS-WINDOWS  system
       it is "C:\latex\LatexMk".

       2) The user's RC file in "$HOME/.latexmkrc".

       3)  The  RC  file  in the current working directory called
       "latexmkrc".

       4) The RC file specified on the command line with  the  -r
       option.
```

It could also be due to the flags that latex-plus uses. I don't know for certain but perhaps an rc file gets overridden by flags passed in on the terminal. Implementing this should be fairly easy. First we would have to detect if a latexmkrc exists in the current directory and then disable all the settings that latex-plus configures.

**Comment by dpo** [comment]
@andrewrynhard Thanks for the fast reply. I'm on OSX and my `latexmk` man page says

```
3)  The  RC  file in the current working directory.  This file can be named
       either "latexmkrc" or ".latexmkrc", and the first of these to be  found  is
       used, if any.
```

so `.latexmkrc` seems ok. However I tried `latexmkrc` without the leading dot, and I also tried placing the RC file inside `.latex` and saw no improvement.

The only place where I see `latexmk` in the code is [in `main.js`](https://github.com/andrewrynhard/atom-latex-plus/blob/master/lib/main.js#L37). The `latexmk` options seem to be in [`compiler.js`](https://github.com/andrewrynhard/atom-latex-plus/blob/master/lib/compiler.js#L44).

The simplest might be to add a config variable for `latexmk` options beyond `-interaction=nonstopmode -f -cd -file-line-error`. That would allow the user to specificy, e.g., `-r /my/latexmkrc`. I tried replacing `-pdf` with `-pdfps` and now `latex` is invoked instead of `pdflatex`.

(On a side note, `xelatex` and `lualatex` could be used simply by passing `-xelatex` or `-lualatex` to `latexmk`, and the `texProgram` option wouldn't really be necessary.)

I'm still having troubles but they seem to be due to something else.
